### PR TITLE
Update preview link

### PR DESCRIPTION
**Pull request by Natsvora**
Update preview link

### CODE REVIEW

**Comment by dambaev** [comment]
@Natsvora : it seems, that this PR includes changes from #67 PR . Current PR named 'update preview link', but it contains much more changes, which seems like a mistake for me?

**Review by dambaev — CHANGES_REQUESTED** [review]
it seems, that this PR includes changes from https://github.com/op-energy-foundation/op-energy-blockspan-service/pull/67 PR . Current PR named 'update preview link', but it contains much more changes, which seems like a mistake for me?

**Comment by Natsvora** [comment]
> it seems, that this PR includes changes from #67 PR . Current PR named 'update preview link', but it contains much more changes, which seems like a mistake for me?

Reason for this is I have created update preview link from 67 branch so once that is merged we will rebased it or else we can discard 67 and just use 68 only. what do you suggest?

**Comment by dambaev** [comment]
okay, #67 had been approved, you can merge it and rebase on it

**Comment by Natsvora** [comment]
> okay, #67 had been approved, you can merge it and rebase on it

Can you please check now?

File: frontend/src/app/oe/components/block-rate-strike-details-v2/block-rate-strike-details-v2.component.ts [hunk 1]
```diff
@@ -80,8 +79,8 @@ export class BlockRateStrikeDetailsV2Component implements OnInit {
               (strikeHeight - this.latestBlock.height) * 600;
           }
 
-          if (fromBlockHeight > strikeHeight) {
-            fromBlockHeight = strikeHeight - 13;
+          if (!fromBlockHeight) {
+            fromBlockHeight = strikeHeight - 14;
```
Comment (dambaev) [inline frontend/src/app/oe/components/block-rate-strike-details-v2/block-rate-strike-details-v2.component.ts:83]: looks like we need
```
fromBlockHeight = max (0, strikeHeight - 14);
```
here in order to not to work with negative fromBlockHeight?

**Review by dambaev — CHANGES_REQUESTED** [review]

File: frontend/src/app/oe/components/block-rate-strike-details-v2/block-rate-strike-details-v2.component.ts (hunk 1 above)
Comment (Natsvora) [reply frontend/src/app/oe/components/block-rate-strike-details-v2/block-rate-strike-details-v2.component.ts]: Done

**Pull request merged**